sips/minutes/_posts/2017-02-14-sip-minutes.md

```diff
@@ -0,0 +1,246 @@
+---
+layout: sip-landing
+title: SIP Meeting Minutes - 14th February 2017
+---
+
+# Minutes
+
+The following agenda was distributed to attendees:
+
+| Topic | Reviewer |
+| --- | --- |
+| [SIP-XX: Improving binary compatibility with @stableABI](http://docs.scala-lang.org/sips/pending/binary-compatibility.html) | Dmitry Petrashko |
+| [SIP-NN - Allow referring to other arguments in default parameters](http://docs.scala-lang.org/sips/pending/refer-other-arguments-in-args.html) | Pathikrit Bhowmick |
+| [SIP 25 - @static fields and methods in Scala objects(SI-4581)](http://docs.scala-lang.org/sips/pending/static-members.html) | Dmitry Petrashko, Sébastien Doeraene and Martin Odersky |
+| [SIP-NN - Match infix & prefix types to meet expression rules](http://docs.scala-lang.org/sips/pending/make-types-behave-like-expressions.html) | Oron Port |
+
+Jorge Vicente Cantero was the Process Lead and acting secretary of the meeting.
+
+## Date, Time and Location
+
+The meeting took place at 5:00pm Central European Time / 8:00am Pacific Daylight
+Time on Tuesday, 14th February 2016 via Google Hangouts.
+
+Minutes were taken by Jorge Vicente Cantero, acting secretary.
+
+## Attendees
+
+Attendees Present:
+
+* Martin Odersky ([@odersky](https://github.com/odersky)), EPFL
+* Dmitry Petrashko (@DarkDimius)(https://github.com/DarkDimius), EPFL
+* Lukas Rytz ([@lrytz](https://github.com/lrytz)), Lightbend
+* Seth Tisue ([@SethTisue](https://github.com/SethTisue)), Lightbend
+* Iulian Dragos ([@dragos](https://github.com/dragos)), Independent
+????* Heather Miller ([@heathermiller](https://github.com/heathermiller)), Scala Center
+* Sébastien Doeraene ([@sjrd](https://github.com/sjrd)), EPFL
+* Eugene Burmako ([@xeno-by](https://github.com/xeno-by)), EPFL
+* Jorge Vicente Cantero ([@jvican](https://github.com/jvican)), Process Lead
+?????* Adriaan Moors ([@adriaanm](https://github.com/adriaanm)), Lightbend
+
+## Proceedings
+
+### Opening Remarks
+
+### SIP 25 - @static fields and methods in Scala objects(SI-4581)
+Jorge asks Dmitry to explain the biggest changes since the last proposal.
+The biggest changes were addressing the feedback of reader.
+**Dmitry** says which included three questions. first cover whether the static can be tail ____ annotation (1:30) which doesn't actually impact compilation but only warns if it isn't possible to make something static. I don't think so because static effects binary compatibility. Also, it depends on the superclass. If the superclass defined a field with the same name, the subclass can't have it. If we decide to make something static, we only be able to do it if the superclass doesn't have the fields with the same name. If static is done automatically and not triggered by the user we will enforce very strong requirements on superclass objects which is no-go.
+
+I proposed a scheme (which I elaborate more on the details frictions in the SIP). I present several examples of possible issues in this case.
```

Comment:
"Details frictions" didn't make sense to me but that's what I heard when i replayed it a few times. This one was hard to summarize but the last two were easier.